build.gradle.kts

```diff
@@ -13,18 +12,18 @@ import org.gradle.api.plugins.JavaPlugin.COMPILE_ONLY_API_CONFIGURATION_NAME
 import org.gradle.api.plugins.JavaPlugin.TEST_IMPLEMENTATION_CONFIGURATION_NAME
 
 plugins {
-    val indraVersion = "1.3.1"
+    val indraVersion = "2.0.0"
     id("net.kyori.indra") version indraVersion apply false
     id("net.kyori.indra.checkstyle") version indraVersion apply false
-    id("net.kyori.indra.publishing.sonatype") version indraVersion apply false
+    id("net.kyori.indra.publishing.sonatype") version indraVersion
```

Comment:
why was this changed?

Reply:
gradle publishing plugin requires being applied to the root project